pkgs/development/python-modules/urllib3/default.nix

```diff
@@ -29,8 +32,9 @@ buildPythonPackage rec {
   };
 
   # FIXME: remove backwards compatbility hack
```

Comment:
```suggestion
```
We should rather make both optional-dependencies opt-in. Since this will probably cause a massive fallout on linux this needs to go to staging or much preferable python-updates. But we currently don't have any python-updates run running, so this presumable needs to wait a little bit.